### DISCUSSION TITLE
ya having two databasesschemas are not [Sep 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3921] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **twarogowski**  
#### both tables sit in one DB, only in different schemas (for security reasons). Right now I have a custom app for handling that (express.js + vue.js)  
#  
> 🙋‍♂️ **twarogowski**  
#### another questionL: can I achieve that by tinkering with nocodb internal database? I store nocodb metadata in Postgres and I can manually edit it (at least theoretically)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1021403471892729938)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
